cloudbuild.yaml

```diff
@@ -374,6 +374,23 @@ steps:
     dir: cmd/sdk-server/bin
     args: [-m, cp, '*.zip', gs://agones-artifacts/sdk-server]
 
+  #
+  # Cleanup services "preview", "development", "default"
+  #
+  - name: gcr.io/cloud-builders/gcloud
```

Comment:
I would add `allowFailure: true` here. On occasion we're going to have a race where two builds run and try to delete the same previous version, but it doesn't matter. Today we run this on an instance and probably never see logs - adding `allowFailure: true` will at least mean we see that a step failed, but not fail the whole build.